src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const lines = sourceString.split(';').filter((line) => line.trim() !== '');
+
+  lines.forEach((line) => {
+    const parts = line.split(':');
```

Comment:
```suggestion
    const [property, value] = line.split(':');
```
You can use destructing